src/map/Map.js

```diff
@@ -74,6 +74,18 @@ export var Map = Evented.extend({
 		// or `L.Canvas` by default depending on browser support.
 		renderer: undefined,
 
+		// @option gradients: []
+		// SVG gradient definitions
+		// [{
+		// 	name: 'red-to-green',
+		// 	direction: 'up',
+		// 	stops: [
+		// 		{offset: 0, color: 'red'},
+		// 		{offset: 100, color: 'green'},
+		// 	]
+		// }]
+		gradients: [],

```

Comment:
No, SVG gradients should be specified on a per-`L.SVG` basis instead. That would allow a modicum of flexibility when it comes to deleting gradients that are no longer needed (e.g. removing a no-longer-needed `L.SVG` from the map), and keeps the scope of the `L.Map` cleaner (edge case where Leaflet is built on ES6 modules with just Canvas support and without SVG support).